model.js

```diff
@@ -55,14 +57,28 @@ StoreModel.prototype.cookiesPerDay = function ()
     calculatedSum = (valueFromArray + calculatedSum);
   }
   this.simulatedCookiesByDay = calculatedSum;
-};
+} ;
+
+var firstAndPike = new StoreModel('First And Pike', 23, 65, 6.3, [6,7,8,9,10,11,12,13,14,15,16,17,18,19,20]);
+
+var seaTacAirport = new StoreModel('SeaTac Airport', 3, 24, 1.2, [6,7,8,9,10,11,12,13,14,15,16,17,18,19,20]);
+
+var seattleCenter = new StoreModel('Seattle Center', 11, 38, 3.7, [6,7,8,9,10,11,12,13,14,15,16,17,18,19,20]);
+
+var capitolHill = new StoreModel('Capitol Hill', 11, 38, 2.3, [6,7,8,9,10,11,12,13,14,15,16,17,18,19,20]);
+
+var alki = new StoreModel('Alki', 2, 16, 4.6, [6,7,8,9,10,11,12,13,14,15,16,17,18,19,20]);
 
 var listOfStores = [
-  new StoreModel('First And Pike', 23, 65, 6.3),
-  new StoreModel('SeaTac Airport', 3, 24, 1.2),
-  new StoreModel('Seattle Center', 11, 38, 3.7),
-  new StoreModel('Capitol Hill', 11, 38, 2.3),
-  new StoreModel('Alki', 2, 16, 4.6),
+  new StoreModel('First And Pike', 23, 65, 6.3, [6,7,8,9,10,11,12,13,14,15,16,17,18,19,20]),
+  new StoreModel('SeaTac Airport', 3, 24, 1.2, [6,7,8,9,10,11,12,13,14,15,16,17,18,19,20]),
+  new StoreModel('Seattle Center', 11, 38, 3.7, [6,7,8,9,10,11,12,13,14,15,16,17,18,19,20]),
+  new StoreModel('Capitol Hill', 11, 38, 2.3, [6,7,8,9,10,11,12,13,14,15,16,17,18,19,20]),
+  new StoreModel('Alki', 2, 16, 4.6, [6,7,8,9,10,11,12,13,14,15,16,17,18,19,20]),
```

Comment:
Since `hours` is the same across all `StoreModel`s, I would set it on the `prototype` instead of accepting it as a parameter.